### PR TITLE
FIX: slow mode dialog doesn't remember Enabled Until value

### PR DESCRIPTION
**Pull request by AndrewPrigorshnev**
If reload a page after enabling slow mode and open the slow mode dialog again it would show a slow mode interval but wouldn't show Enabled Until value.

This PR fixes it.

To be merged after[ FIX: backtracking re-render" assertion in future-date-input](https://github.com/discourse/discourse/pull/13075). That PR will fix an ember assertion and make the test in this PR pass.

### CODE REVIEW

File: app/assets/javascripts/discourse/app/controllers/edit-slow-mode.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ export default Controller.extend(ModalFunctionality, {
   minutes: null,
   seconds: null,
   saveDisabled: false,
-  enabledUntil: null,
+  enabledUntil: alias("model.slow_mode_enabled_until"),
```
Comment (riking) [inline app/assets/javascripts/discourse/app/controllers/edit-slow-mode.js:17]: can we instead change the binding at the point of usage where it's written to? (It's probably in the .hbs file.)

File: app/assets/javascripts/discourse/tests/acceptance/topic-slow-mode-test.js [hunk 2]
```diff
@@ -0,0 +1,560 @@
...
+import I18n from "I18n";
+
+acceptance("Topic - Slow Mode - enabled", function (needs) {
+  needs.user();
+  needs.pretender((server, helper) => {
+    server.get("t/94.json", () => {
+      const slowModeSeconds = 600;
+      const slowModeEnabledUntil = "2040-01-01T04:00:00.000Z";
+
+      return helper.response({
```
Comment (riking) [inline app/assets/javascripts/discourse/tests/acceptance/topic-slow-mode-test.js:17]: Consider importing an existing fixture and patching up the fields that differ.

**Review by riking — APPROVED** [review]
Some nits, but the code should function as is.

**Comment by AndrewPrigorshnev** [comment]
@riking good suggestions 👍 thank you!

**Pull request merged**